### PR TITLE
RS-269: Move quick start guides into main documentation

### PR DESCRIPTION
**Pull request by atimin**
* Move Python, JavaScript, C++ Quick Start Guides
* Write Rust Guide
* Create components for section to avoid duplication

### CODE REVIEW

File: docs/getting-started/common/data_ingestion.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export default function DataIngestionSection(props: Readonly<Props>): JSX.Element {
+    const children = props.children;
+    return (
+        <div>
+            <p>
+                Time series data is stored in entries within a bucket. An entry is a collection of records with unique
+                timestamps. It must have a unique name within the bucket and usually represents a data source,
+                such as <b>
+                <a href="src/pages/use-cases/vibration-sensors/index.tsx">a vibration sensor</a>
```
Comment (AnthonyCvn) [inline docs/getting-started/common/data_ingestion.tsx:15]: Got issue with the link. Try:

```javascript
import Link from '@docusaurus/Link';

<Link to="/use-cases/vibration-sensors">a vibration sensor</Link>
```

File: docs/getting-started/common/data_ingestion.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            </b> or <b>
+                <a href="/blog/2024-02-16-3-ways-stora-data-for-computer-vision-applications/index.mdx">a CV
```
Comment (AnthonyCvn) [inline docs/getting-started/common/data_ingestion.tsx:17]: Also try with import Link from '@docusaurus/Link';

File: docs/getting-started/with-python.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
+---
+title: Quick Start With Python Client SDK
```
Comment (AnthonyCvn) [inline docs/getting-started/with-python.mdx:2]: The title "Quick Start With Python" could be enough (same for the other languages)

**Review by AnthonyCvn — CHANGES_REQUESTED** [review]
Looks really good! I have small suggestions in the comment, and should the pages already be under `/docs/getting-started`? In that case we also need to add the canonical links to avoid duplicate content.

**Comment by atimin** [comment]
@AnthonyCvn  I'll release 1.10 version when we merge the PR. All the canonical links must work afterwards.

**Pull request merged**